cogs/annual_handover_and_reset.py

```diff
@@ -0,0 +1,225 @@
+            for channel in category.channels:
+                logger.debug("Resetting channel permissions for channel: %s", channel)
+                await channel.set_permissions(committee_elect_role, overwrite=None)
+
+        await initial_response.edit(
+            content=(
+                ":hourglass: Giving committee users access to the #handover channel and "
+                "removing the \"Committee\" role... :hourglass:"
+            ),
+        )
+
+        handover_channel: discord.TextChannel | None = discord.utils.get(
+            main_guild.text_channels,
+            name="Handover",
+        )
+
+        automod_role: discord.Role | None = discord.utils.get(
+            main_guild.roles,
+            name="Automod",
+        )
+
+        committee_member: discord.Member
+        for committee_member in committee_role.members:
+            if committee_member.bot:
+                continue
+            if handover_channel:
+                logger.debug("Giving user: %s, access to #handover", committee_member)
+                await handover_channel.set_permissions(
+                    committee_member,
+                    read_messages=True,
+                    send_messages=True,
+                )
+
+            logger.debug("Removing Committee role from user: %s", committee_member)
+            await committee_member.remove_roles(
+                committee_role,
+                reason=f"{ctx.user} used TeX-Bot slash-command: \"handover\"",
+            )
+
+            if automod_role and automod_role in committee_member.roles:
+                await committee_member.remove_roles(
+                    automod_role,
+                    reason=f"{ctx.user} used TeX-Bot slash-command: \"handover\"",
+                )
+
+        await initial_response.edit(
+            content=(
+                ":hourglass: Giving committee-elect users the \"Committee\" role "
+                "and removing their \"Committee-Elect\" role... :hourglass:"
+            ),
+        )
+
+        committee_elect_member: discord.Member
+        for committee_elect_member in committee_elect_role.members:
+            if committee_elect_member.bot:
+                continue
+            logger.debug("Giving user: %s, the committee role.", committee_elect_member)
+            await committee_elect_member.add_roles(
+                committee_role,
+                reason=f"{ctx.user} used TeX-Bot slash-command: \"handover\"",
+            )
+
+            logger.debug("Removing Committee-Elect role from user: %s", committee_elect_member)
+            await committee_elect_member.remove_roles(
+                committee_elect_role,
+                reason=f"{ctx.user} used TeX-Bot slash-command: \"handover\"",
+            )
+
+        await initial_response.edit(content=":white_check_mark: Handover procedure complete!")
+
+
+class AnnualResetRolesCommandCog(TeXBotBaseCog):
+    """Cog class that defines the reset_roles command."""
```

Comment:
Use updated name, speech marks & forward-slash: `...the "/annual-roles-reset" command."""`